examples/with-nestjs/packages/api/package.json

```diff
@@ -41,7 +41,7 @@
     "@repo/typescript-config": "workspace:*",
     "@types/node": "^20.3.1",
     "ts-loader": "^9.4.3",
-    "ts-node": "^10.9.1",
+    "ts-node": "10.9.2",
```

Comment:
did you mean to remove the `^`?

Reply:
truth be told, yes. but it's a bad habit I'm trying to break (my heart is ideologically incorrect so I often remove them: have mercy on me lol)

fixed in 8fcb086b6a033ce28bb046b863a2dc87e46898c2

Reply:
(and updated lockfile in 51f4a3ba80f9cf5532268d0b58fe06473d7c18d4)